docs/content/integrations/airflow.mdx

```diff
@@ -7,213 +7,36 @@ description: The dagster-airflow package allows you to export Dagster jobs as Ai
+| Executor              | [Executor](/deployment/executors)                                                                                                                                                                                                                                                                      |                                                                                                                                                                                                       |
+| DagBag                | [Code Locations](/concepts/code-locations)                                                                                                                                                                                                                                                             | Multiple isolated code locations with different system and python dependencies can exist within the same dagster instance                                                                             |
+| Instance              | [Instance](/deployment/dagster-instance)                                                                                                                                                                                                                                                               |                                                                                                                                                                                                       |
+| SubDAGs / TaskGroups  | [Graphs](/concepts/ops-jobs-graphs/graphs), [Tags](/concepts/ops-jobs-graphs/metadata-tags#tags) and [AssetGroups](/concepts/assets/software-defined-assets#grouping-assets)                                                                                                                           | Dagster provides rich, searchable [metadata and tagging](/concepts/ops-jobs-graphs/metadata-tags) support well beyond what’s offered by Airflow.                                                      |
+| Hooks                 | [Resources](/concepts/resources)                                                                                                                                                                                                                                                                       | Dagster [resources](/concepts/resources) contain a superset of the functionality of hooks and have much stronger composition guarantees;                                                              |
```

Comment:
```suggestion
| Hooks                 | [Resources](/concepts/resources)                                                                                                                                                                                                                                                                       | Dagster [resources](/concepts/resources) contain a superset of the functionality of hooks and have much stronger composition guarantees.                                                              |
```

Removing the semicolon and adding a period to make this consistent with how other notes in the table are formatted (ending with periods)